librz/core/creg.c

```diff
@@ -0,0 +1,96 @@
+// SPDX-FileCopyrightText: 2021 Florian Märkl <info@florianmaerkl.de>
+// SPDX-FileCopyrightText: 2021 ret2libc <sirmy15@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_core.h>
+
+/**
+ * \brief Get the currently relevant RzReg
+ *
+ * Depending on whether or not the core is in debug mode, this returns the RzReg
+ * from debug or analysis (emulation).
+ * Before using this function, think twice whether it would not make more sense
+ * to use core->dbg->reg or analysis->dbg->reg directly, depending on what you want
+ * to do with it.
+ */
+RZ_API RzReg *rz_core_reg_default(RzCore *core) {
+	return rz_core_is_debug(core) ? core->dbg->reg : core->analysis->reg;
+}
+
+/// rz_reg_getv_by_role_or_name() on rz_core_reg_default()
```

Comment:
is `///` alright?

shouldn't it be:
```C
/**
  * xxxxxx
  */
```
?

Reply:
Just re-checked and you are right. Seems like I have been doing this wrong for quite a while...

Reply:
To be exact: just using `///` works if there are at least two lines. With one it does not. Documented here: https://www.doxygen.nl/manual/docblocks.html#cppblock